### PR TITLE
Implement pending fixes

### PR DESCRIPTION
**Pull request by LegenJCdary**

### CODE REVIEW

File: ansible_deployer/command_line.py [hunk 1]
```diff
@@ -71,8 +71,8 @@ def parse_options(argv):
                         help='Print debug output.')
     parser.add_argument("--syslog", "-v", default=False, action="store_true", help='Log warnings '
                         'and errors to syslog. --debug doesn\'t affect this option!')
-    parser.add_argument("--limit", "-l", nargs=1, default=[None], metavar="[LIMIT]",
-                        help='Limit task execution to specified host.')
+    parser.add_argument("--limit", "-l", nargs='+', default=[], metavar="LIMIT_1 LIMIT_2",
```
Comment (cinek810) [inline ansible_deployer/command_line.py:74]: If we're going to add this commit we need to add test of functionality too. Is that realy required - won't just running host1,host2 work? If we don't need that I vote to drop it.

File: ansible_deployer/command_line.py [hunk 2]
```diff
@@ -367,7 +367,8 @@ def get_playbooks(config: dict, options: dict):
     for play in play_names:
         for item in config["tasks"]["play_items"]:
             if item["name"] == play:
-                if item["skip"]:
+                skip = item.get("skip", [])
```
Comment (cinek810) [inline ansible_deployer/command_line.py:370]: Does that fix something? if/for sequence above is just different

File: ansible_deployer/command_line.py [hunk 3]
```diff
@@ -504,7 +504,7 @@ def get_tags_for_task(config: dict, options: dict):
     tags = []
     for task in config["tasks"]["tasks"]:
         if task["name"] == options["task"]:
-            tags = task["tags"]
+            tags = task.get("tags", [])
```
Comment (cinek810) [inline ansible_deployer/command_line.py:507]: this commit looks good, but it's not a separate PR..

**Review by cinek810 — CHANGES_REQUESTED** [review]

File: ansible_deployer/command_line.py (hunk 3 above)
Comment (LegenJCdary) [reply ansible_deployer/command_line.py:507]: Moved to #50

File: ansible_deployer/command_line.py (hunk 2 above)
Comment (LegenJCdary) [reply ansible_deployer/command_line.py:370]: Moved to #50 
It changes handling of field "skip" in tasks config. If there is no such field (we agreed on it being optional) empty list will be assigned and following condition will not be checked.

**Pull request closed without merging**